frontend/src/pages/register.tsx

```diff
@@ -69,6 +69,10 @@ export const RegisterPage: NextPage = () => {
     )
   }, [username, password, displayName, passwordAgain])
 
+  const isWeekPassword = useMemo(() => {
```

Comment:
```suggestion
  const isWeakPassword = useMemo(() => {
```

Reply:
Thanks @DerMolly  for the review. Made the changes